pyspark/bigdl/optim/keras/training.py

```diff
@@ -0,0 +1,44 @@
+#
+# Copyright 2016 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from bigdl.util.common import *
+
+
+class Training(JavaValue):
+    def __init__(self, bigdl_type="float"):
+        self.bigdl_type = bigdl_type
+
+    def compile(self, optimizer, loss, metrics=None):
```

Comment:
Should they keep the same hierarchy same as Scala? i.e put them into "KerasLayer"

Reply:
Maybe "KerasLayer" also not a good place for this as ideally we only allow "Model" and "Sequential" to have compile and fit.